app/controllers/import_controller.rb

```diff
@@ -278,6 +279,17 @@ def import_date_field(value)
     value.blank? ? nil : value
   end
 
+  def import_time_field(field, value, row_ind)
+    return nil if value.blank?
+
+    normalized_value = value.downcase.strip.gsub(/[ -.]/, '')
```

Comment:
I tried some garbage data for things that could be typos or typed due to misunderstanding and there are some cases that don't provide a useful error messages

# Setting Preferred Contact Time to `16`
![Screen Shot 2021-08-01 at 10 18 12 AM](https://user-images.githubusercontent.com/24628687/127778035-413a0f07-4993-4c65-862d-010cf0edd047.png)

# Setting Preferred Contact Time to `0:00`
![Screen Shot 2021-08-01 at 10 19 55 AM](https://user-images.githubusercontent.com/24628687/127778079-e7b625cc-432e-4c80-b385-27d0aafa3532.png)

# Setting Preferred Contact Time to `24:00`
![Screen Shot 2021-08-01 at 10 20 46 AM](https://user-images.githubusercontent.com/24628687/127778107-28b7a880-61c0-4550-9027-8ba413ec1990.png)

Reply:
hmm the error messages I'm seeing seem to be different?

![Screen Shot 2021-08-02 at 11 14 10 AM](https://user-images.githubusercontent.com/9956561/127884132-8deadc73-61bf-4550-867a-9deefdf46f68.png)

Reply:
It might be due to the data type that is in the excel sheet for those specific cells

Reply:
ah got it, thanks! https://github.com/SaraAlert/SaraAlert/pull/1019/commits/36a8a570ebaaf09f77ca616a821f7d03b929db82